dts/arm/nxp/nxp_rt1064.dtsi

```diff
@@ -10,7 +10,7 @@
 	reg = <0x402a4000 0x4000>, <0x70000000 0x400000>;
 	/* WINBOND */
 	w25q32jvwj0: w25q32jvwj@0 {
-		compatible = "winbond,w25q32jvwj", "jedec,spi-nor";
+		compatible = "winbond,w25qxx", "jedec,spi-nor";
```

Comment:
This should be: 

compatible = "winbond,w25q32jvwj", "winbond,w25qxx", "jedec,spi-nor";

Reply:
Oddly, I didn't find "winbond, w25q32jvwj" elsewhere, why this configuration will take effect,
I just use w25q for global search, see where I used it, and modify it.
If this has nothing to do with this driver, you can restore it here.

Reply:
Winbond W25 devices conform to the JEDEC SPI NOR API, so the standard `CONFIG_SPI_NOR=y` should be used to communicate with them.  That's the only way this could have worked before.

Switching to a different device does not seem correct.